.github/workflows/codeQL-scanning.yaml

```diff
@@ -0,0 +1,49 @@
+name: "CodeQL scanning"
+
+on:
+  push:
+    branches:
+      - master
+      - 'release/*'
+  pull_request:
+    branches:
+      - main
+      - 'release/*'
+    paths:
+      - 'node/**'
+      - 'operators/**'
+      - 'retriever/**'
+      - 'disperser/**'
+      - 'core/**'
+      - 'common/**'
+      - 'api/**'
+      - '.github/codeql/**'
+      - '.github/workflows/codeql-analysis.yml'
+  schedule:
+    - cron: '0 9 * * *'
```

Comment:
It's triggered already on push, what this cron test adds?

Reply:
adds a cadence of running the codeql scanner everyday at 9 am. Helps keep a regular non PR triggered scan of the code. Idea is to use this for longer and slower codeql scans. Will be adding that soon-ish.